content/en/users/aai/check-in/vos/_index.md

```diff
@@ -435,44 +435,44 @@ be supported in the future.
 
 #### VO Groups Methods
 
-1. Retrieving all VO groups:
+##### Retrieving all VO groups
```

Comment:
Retrieving all VO groups details. 
IS it possible to add another API example: Retrieving all VO groups details managed by a given VO Manager?

Reply:
Closing this to merge this PR that is now approved. Please add it to another PR if needed.